source/replication.rst

```diff
@@ -0,0 +1,220 @@
+* ``votes`` key sets the number of `votes <https://docs.mongodb.com/manual/reference/replica-configuration/#mongodb-rsconf-rsconf.members-n-.votes>`_
+  an external node can cast in a replica set election (``0`` by default, and
+  ``0`` for non-voting members of the cluster). 
+
+Here is an example:
+
+.. code:: yaml
+
+   spec:
+     unmanaged: false
+     replsets:
+     - name: rs0
+       externalNodes:
+       - host: rs0-1.percona.com
+         port: 27017
+         priority: 0
+         votes: 0   
+       - host: rs0-2.percona.com
+       ...
+     sharding:
+       configsvrReplSet:
+         size: 3
+         externalNodes:
+           - host: cfg-1.percona.com
+             port: 27017
+             priority: 0
+             votes: 0   
+           - host: cfg-2.percona.com
+           ...
+
+The *Main* cluster will be ready for replication when you apply changes as usual:
+
+.. code:: bash
+
+   $ kubectl apply -f deploy/cr.yaml
+
+.. _operator-replication-source-secrets:
+
+Getting the Main cluster secrets and certificates to be copied to Replica
+*************************************************************************
+
+*Main* and *Replica* cluster should have same Secrets objects (to have same
+users credentials) and certificates. So you may need to copy them from your
+*Main* cluster. Names of the corresponding objects are set in the ``users``,
+``ssl``, and ``sslInternal`` keys of the Custom Resource ``secrets`` subsection
+(``my-cluster-name-secrets``, ``my-cluster-name-ssl``, and
+``my-cluster-name-ssl-internal`` by default).
+
+If you can get Secrets from an existing cluster by executing the
+``kubectl get secret`` command for *each* Secrets object you want to acquire:
+
+.. code:: bash
+
+   $ kubectl get secret my-cluster-name-secrets -o yaml > my-cluster-secrets.yaml
+
+Next remove the ``annotations``, ``creationTimestamp``, ``resourceVersion``,
+``selfLink``, and ``uid`` metadata fields from the resulting file to make it
+ready for the *Replica* cluster.
+
+.. _operator-replication-replica:
+
+Configuring cross-site replication on Replica instances
+-------------------------------------------------------
+
+When the Operator creates a new cluster, a lot of things are happening, such as
+electing the Primary, generating certificates, and picking specific names. This
+should not happen if we want Operator to run the cluster as *Replica*, so first
+of all the cluster should be put into unmanaged state by setting the
+``unmanaged`` key in the ``deploy/cr.yaml`` configuration file to true.
+
+.. note:: Setting ``unmanaged`` to true will not only prevent the Operator from
+   controlling the Replica Set configuration, but it will also result in not
+   generating certificates and users credentials for new clusters.
+
+The cluster should also "know" about external nodes for the Replica Sets. You
+can configure this information in the ``replsets.externalNodes`` and
```

Comment:
know and information are killing the vibe

Reply:
user does not need to add external nodes on replica at all. not for config servers, not for replsets.